packages/aws-cdk-lib/aws-dynamodb/TABLE_V1_API.md

```diff
@@ -273,4 +273,28 @@ new dynamodb.Table(this, 'MyTable', {
 });
 ```
 
-If you have a global table replica, note that it does not support the addition of a resource-based policy.
\ No newline at end of file
+If you have a global table replica, note that it does not support the addition of a resource-based policy.
```

Comment:
Any idea if this statement apply to the steam resource-based policy as well? If yes, we should update this.